.gitignore

```diff
@@ -28,3 +28,7 @@ docs/_static/css/eprosima_rtd_theme.css
 build/
 install/
 log/
+
```

Comment:
I think that @EduPonz is not a great fan of adding to the .gitignore file. I am not against it :sweat_smile:

Reply:
As long as @EduPonz does not see it...